index.js

```diff
@@ -62,30 +62,44 @@ var DetectBrowsers = function (config, logger) {
 
     config = config || {};
     config.detectBrowsers = config.detectBrowsers || {};
+    config.plugins = config.plugins || [];
 
     if (config.detectBrowsers.enabled === false) {
         log.info('Detecting browsers is disabled. The browsers of the browsers array are used.');
         return;
     }
 
-    var availableBrowser = getInstalledBrowsers(browsers);
+    const configuredLaunchers = config.plugins.reduce((prev, plugin) => {
```

Comment:
Can we use es5 syntax please. Just to ensure backwards compatibility.

Reply:
Ah, sorry. I caught myself using es6 a couple times, missed this one. It's fixed now.